apps/vdjbase-singularity/0.1/cedar/upload-bundle.sh

```diff
@@ -0,0 +1,23 @@
+#
+TOOL=changeo-singularity
+SYSTEM=ls5
```

Comment:
cedar?

Reply:
I don't use this file in the current App, I am using Git to stage changes to the deployment directory on cedar. It should be changed, yes...

Reply:
Fixed on my latest commit, although still not using these files.